web/app/configs/lang.json

```diff
@@ -141,9 +141,17 @@
         "importfrom": "of import from",
         "productionimport": "production/import",
         "consumption": "consumption",
-        "electricityfrom": "electricity comes from",
-        "electricityto": "electricity is exported to",
-        "electricitystored": "electricity is stored using",
+
+        "electricityComesFrom": "{1} % of electricity in <img id=\"country-flag\"></img> <b>{2}</b> comes from {3}",
```

Comment:
It is enough to have `"{1} % of electricity in {2} comes from {3}"`
`{2}` is then replaced by `<img id=\"country-flag\"></img> <b>{2}</b>`

Or as a alternative use: `"{1} % of electricity in {2} {3} comes from {4}"`
where {2} is replaced by `<img id=\"country-flag\"></img>` 
and {3} is replaced by `<b>{2}</b>`

In that way it is easier to translate and easier to change the html without to having to change each translation.